src/Future.js

```diff
@@ -35,12 +35,12 @@ Future.prototype.ap = function(m) {
       }
     }
 
-    self.fork(doReject, function(fn) {
+    self._fork(doReject, function(fn) {
       applyFn = fn;
       resolveIfDone();
     });
 
-    m.fork(doReject, function(v) {
+    m._fork(doReject, function(v) {
```

Comment:
The consequence of this line, I just realized, is that it killed interoperability with other potential Futures that implement the "*standard*" of a `.fork(rej, res)`. I'm not sure this is of too big a concern, but it might annoy someone somewhere.